test/ANTLR4UtilTests.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (May of 2023,
+// schlegea@informatik.uni-freiburg.de)
+
+#include <ANTLRInputStream.h>
+#include <gtest/gtest.h>
+
+#include "../benchmark/infrastructure/util/ANTLRLexerHelper.h"
+#include "../test/util/ANTLR4Mockup/ANTLR4MockupLexer.cpp"  // The mockup lexer is used NOWHERE else, so this should be alright.
```

Comment:
Still don't do this.

Reply:
I didn't want to destroy the whole test setup for one specific case, which is why I did it like this.